cmake/kokkos_enable_options.cmake

```diff
@@ -52,7 +52,11 @@ KOKKOS_ENABLE_OPTION(EXPLICIT_INSTANTIATION   OFF
   "Whether to explicitly instantiate certain types to lower future compile times")
 SET(KOKKOS_ENABLE_ETI ${KOKKOS_ENABLE_EXPLICIT_INSTANTIATION} CACHE INTERNAL "eti")
 
-IF (DEFINED CUDA_VERSION AND CUDA_VERSION VERSION_GREATER "7.0")
+IF (KOKKOS_ENABLE_CUDA)
+  SET(KOKKOS_COMPILER_CUDA_VERSION "${KOKKOS_VERSION_MAJOR}${KOKKOS_VERSION_MINOR}")
```

Comment:
Can't help myself from ranting about this silly naming `KOKKOS_VERSION_{MAJOR,MINOR,PATCH}` that does not refer to the Kokkos version but to the compiler version :@

Reply:
wait this is weird where is that coming from? I though KOKKOS_VERSION is actually the version of Kokkos??